src/uu/head/src/head.rs

```diff
@@ -243,12 +243,23 @@ fn read_n_lines(input: &mut impl std::io::BufRead, n: u64, separator: u8) -> std
     Ok(())
 }
 
-fn read_but_last_n_bytes(input: &mut impl std::io::BufRead, n: usize) -> std::io::Result<()> {
+fn read_but_last_n_bytes(input: &mut impl std::io::BufRead, n: u64) -> std::io::Result<()> {
     if n == 0 {
         //prints everything
         return read_n_bytes(input, std::u64::MAX);
     }
 
+    let n = match usize::try_from(n) {
+        Ok(value) => value,
+        Err(e) => {
+            show!(USimpleError::new(
+                1,
+                format!("{e}: number of bytes is too large")
+            ));
+            return Ok(());
+        }
+    };
```

Comment:
can be simplify with something like
```suggestion
let n = usize::try_from(n).map_err(|e| {
    show!(USimpleError::new(1, format!("{e}: number of bytes is too large")));
    e
})?;
```

Reply:
I experimented with it. The problem: The proposed alternative is actually not 100% the same. It makes the function return with an Err(..) (which needs to be converted first). But originally it just prints the error and then returns with Ok(). 

I personally think this is a bit smelly anyway. But i moved this code just from a different location (cleanup commit will come soon).

Reply:
additionally: I actually can't test this part of code. On 64bit machine usize and u64 is the same. Thus this convertion will only fail on 32 bit (or lower if this exists here)